common/libs/units/units.h

```diff
@@ -110,11 +110,31 @@ template <class Q, class ValTy> class Scalar {
   ValTy val_;
 };
 
-// Represents a Scalar where addition and subtraction are well-defined.
+// Represents a Scalar where:
+// - addition and subtraction are well-defined
+// - multiplication by an external factor of type ValTy is defined
```

Comment:
`multiplication and devision by an external factor of type ValTy are defined`

Reply:
OK also added that division needs to be defined on ValTy as well